sherpa/astro/tests/test_astro.py

```diff
@@ -539,6 +539,16 @@ def test_proj(self):
     def test_proj_bubble(self):
         self.run_thread('proj_bubble')
 
+        fit_results = ui.get_fit_results()
+        covarerr = sqrt(fit_results.extra_output['covar'].diagonal())
+        assert covarerr[0] == approx(0, rel=1e-4)
+        assert covarerr[1] == approx(8.74608e-07, rel=1e-3)
+        
+        # Fit -- Results from reminimize
+        assert self.locals['mek1'].kt.val == approx(17.8849, rel=1e-2)
+        self.assertEqualWithinTol(self.locals['mek1'].norm.val, 4.15418e-06,
```

Comment:
Should we have both checks using the same form (preferably the pytest `approx` rather than the `assertEqualWIthinTol` method)? I imagine this would be a clean-up of the original code (ie is not due to the merge issue that caused the tests to be dropped)?

Reply:
Actually, it turns out we have these checks are repeated below  see line 578. However, in that case the tests use an absolute tolerance (was 0.01, now 0.057) rather than a relative one, and an absolute tolerance of ~1e-2 for the normalization, which is ~4e-6, is obviously pointless.

So, I think we can drop this PR but may want to review this test

Reply:
Or perhaps replace the existing tests (e.g. lines 578 and 579) with the "new" ones (lines 548 and 549).

Reply:
I believe I addressed these comments in the new commits (I also rebased to get the latest changes from master).

Reply:
@DougBurke could you please confirm if this is resolved?